docs/shared-content/structured-configuration-variables.include.md

```diff
@@ -173,7 +173,7 @@ foo:
     - qux
 ```
 
-It's possible to replace an entire array or sequence too. With the previous example above, if the Octopus Variable `foo:bar` was to a value of `["baz","qux"]` it would create outputs like:
+It's possible to replace an entire array or sequence too. In the previous example above, if the Octopus Variable `foo:bar` was set to `["baz","qux"]`, it would create outputs like:
```

Comment:
```suggestion
It's possible to replace an entire array or sequence too. In the previous example, if the Octopus Variable `foo:bar` was set to `["baz","qux"]`, it would create outputs like:
```